### PR TITLE
feat: friend request push notifications

### LINKED ISSUE #776: Cancel Friend Request (Push notifications)

### PR DESCRIPTION
**Pull request by Julieta11**
# What? 
This branch continues the work for the New Friend Request feature. Find more info about the proposal [here](https://www.notion.so/decentraland/PRD-New-Friend-Requests-188555225272448f91f3eea5e84f1cd3) and check the RFC [here](https://adr.decentraland.org/adr/ADR-137). **The messages exchange between kernel and renderer is defined by RPC calls.**

So far, the PR addresses:
- Cancel friend request (Push notifications)
- Reject friend request (Push notifications)
- Approve friend request (Push notifications)

### CODE REVIEW

File: packages/shared/friends/sagas.ts [hunk 1]
```diff
@@ -1261,6 +1276,21 @@ function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
...
 
+        // TODO!: remove FF validation once the new flow is the only one
+        if (newFriendRequestFlow && incoming && action === FriendshipAction.REJECTED) {
+          // Build message
+          const rejectFriendRequest = {
+            userId: getUserIdFromMatrix(userId)
+          }
+
+          // Send messsage to renderer via rpc
+          getRendererModules(store.getState())
```
Comment (menduz) [inline packages/shared/friends/sagas.ts:1287]: const rendererModules = yield select(getRendererModules)

File: packages/shared/friends/sagas.ts [hunk 2]
```diff
@@ -1261,6 +1276,21 @@ function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
...
+            ?.friendRequest?.rejectFriendRequest(rejectFriendRequest)
+            .catch((err) => {
```
Comment (menduz) [inline packages/shared/friends/sagas.ts:1289]: yield apply

File: packages/shared/friends/sagas.ts (hunk 1 above)
Comment (Julieta11) [reply packages/shared/friends/sagas.ts:1287]: Oops sorry! I've already merged the branch into another PR. I'll address your comments in that one. Thx 🙌🏼

File: packages/shared/friends/sagas.ts (hunk 2 above)
Comment (Julieta11) [reply packages/shared/friends/sagas.ts:1289]: Same 😛

**Pull request merged**